packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -347,9 +346,7 @@ class HeapTreeViewState extends State<HeapTree>
                     ? 'Graphing...'
                     : _isSnapshotGrouping
                         ? 'Grouping...'
-                        : _isSnapshotComplete
-                            ? 'Done'
-                            : '...',
+                        : _isSnapshotComplete ? 'Done' : '...',
```

Comment:
you'll need to revert this change to get the bots to pass - the failing bot is formatting based on Flutter 1.22.0-12.0.pre

Reply:
Done! My bad, auto-save formatting :)